src/lib.rs

```diff
@@ -199,6 +199,27 @@ impl TreeIndex {
   /// Get the position of the highest entry in the tree. Aka max.
   ///
   /// NOTE: should we rename this to `.len()` ?
+  /// ## Examples
+  /// ```
+  ///        3
+  ///    1       5
+  ///  0   2   4   6  
+  /// ```
+  ///
+  /// ```rust
+  /// let mut tree = TreeIndex::default();
```

Comment:
Tests fail here with `Use of undeclared module TreeIndex`.

![2018-09-03-184015_3840x2160](https://user-images.githubusercontent.com/2467194/44996545-c3414000-afa8-11e8-8345-6eedf075c197.png)

Reply:
ah! fix it.